package.json

```diff
@@ -16,7 +16,8 @@
     "listview"
   ],
   "peerDependencies": {
-    "react-native": ">=0.20.0"
+    "react-native": ">=0.20.0",
```

Comment:
Wanna bump this up to the version that starts requiring `react`? Looks like it was [0.25.1](https://github.com/facebook/react-native/releases/tag/v0.25.1), so `>=0.25.1`